docs/index.md

```diff
@@ -44,84 +44,46 @@
 
 # Welcome to Streamlit
 
-[Streamlit](https://streamlit.io) is an open-source Python library that makes it
-easy to build beautiful custom web-apps for machine learning and data science.
+[Streamlit](https://streamlit.io/) is an open-source Python library that makes it easy to create and share beautiful, custom web apps for machine learning and data science.
+In just a few minutes you can build and deploy powerful data apps - so let's get started!
 
-To use it, run `pip install streamlit` from a terminal, then import it in Python, write a couple lines
-of code, and run your script with `streamlit run [filename]`. Streamlit watches
-for changes on each save and updates the app live while you're coding. Code
-runs from top to bottom, always from a clean state, and with no need for
-callbacks. It's a simple and powerful app model that lets you build rich UIs
-incredibly quickly. To learn more about how Streamlit works, see [Main
-concepts](main_concepts.md).
+1. Make sure that you have [Python 3.6](https://www.python.org/downloads/) or greater installed.
+2. Install Streamlit using [PIP](https://pip.pypa.io/en/stable/installing/):
 
-You may also want to check out [this four-part
-video](https://www.youtube.com/watch?v=R2nr1uZ8ffc&list=PLgkF0qak9G49QlteBtxUIPapT8TzfPuB8)
-recorded at our PyData talk on December 2019. In it we describe the motivation
-behind Streamlit, then go over how to install and create apps with it.
+   `pip install streamlit`
 
-## Install Streamlit
+3. Run the hello world app:
 
-```eval_rst
-.. raw:: html
-
-  <div class='embed-container'><iframe
-    width="560"
-    height="315"
-    src="https://www.youtube.com/embed/sxLNCDnqyFc"
-    style="margin: 0 0 2rem 0;"
-    frameborder="0"
-    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-    allowfullscreen>
-  </iframe></div>
-```
+   `streamlit hello`
 
-Follow these steps and to get a sample app running in less than 5 minutes.
+4. That's it! In the next few seconds the sample app will open in a new tab in your default browser.
 
-1. Make sure that you have [Python 3.6](https://www.python.org/downloads/) or greater installed.
-1. Install Streamlit using [PIP](https://pip.pypa.io/en/stable/installing/):
+Still with us? Great! Now make your own app in just 3 more steps:
 
-   ```bash
-   pip install streamlit
-   ```
+1. Open a new Python file, import Streamlit, and write some code
 
-1. Run the hello world app:
+2. Run the file with:
 
-   ```bash
-   streamlit hello
-   ```
+   `streamlit run [filename]`
 
-1. That's it! In the next few seconds the sample app will open in a new tab in
-   your default browser.
+3. When you're ready, click 'Deploy' from the Streamlit menu and use Streamlit sharing to share your app with the world!
```

Comment:
nit: Remove "and use Streamlit sharing":
- More concise
- Doesn't confuse the user thinking "what, so I need to click Deploy and then do something else???"

Reply:
I think it's really replacing 'and' with 'to'

When you're ready, click 'Deploy' from the Streamlit menu to use Streamlit sharing to share your app with the world!